library/libobject.ml

```diff
@@ -18,11 +18,36 @@ type 'a substitutivity =
 
 type object_name = Libnames.full_path * Names.KerName.t
 
+module NSet = Globnames.ExtRefSet
+
+type open_filter =
+  | Unfiltered
+  | Names of NSet.t
+
+let simple_open f filter i o = match filter with
```

Comment:
How about naming it `unfiltered_open` instead?

Reply:
Sounds a bit too much like an unconditional open, instead of opening only when unfiltered IMO

Reply:
Ok, fair enough.